### ISSUE TITLE
Sending tweet stop working

### ISSUE THREAD

**Issue by madeintheusb** [linvi/tweetinvi#346]
I wrote an app sending tweets with C# that has been working since august 2016,
with tweetinvi 0.9.14.0.

This week I tried it again and though there is no error, the tweet do not show up in my feed.

Any idea?

Thanks.

**Comment by linvi** [comment]
Hello,

> This week I tried it again and though there is no error, the tweet do not show up in my feed.

Are you still using version 0.9.14.0?

If you are, could you please send me the Exception details from the `ExcexptionHandler` : https://github.com/linvi/tweetinvi/wiki/Exception-Handling#global-application-handling-mechanism.

**Comment by madeintheusb** [comment]
The weird thing is that there is no exception at all.
I remember when I just started with tweetinvi, I got exception sending tweet with images.
or in the past if the text message was too long I would get an exception.
But in my current context, I know the tweet I am sending worked in the past.
there is no exception and the tweet does not show up in my twitter feed.

I will re try today, but I am thinking more twitter may have change something.

Fred.
